LICENSE.md

```diff
@@ -0,0 +1,277 @@
+Eclipse Public License - v 2.0
```

Comment:
General comment. The [LICENSE.md](https://github.com/eclipse-thingweb/node-wot/blob/master/LICENSE.md) file for node-wot looks different.

I don't have a strong opinion either way. I just think they should be aligned.

Reply:
Yes that is correct, we should simple copy it over

Reply:
Fixed at [156d7be](https://github.com/eclipse-thingweb/test-things/pull/1/commits/156d7be2cc5b36e9d31e612a4f83c4abedef38fc)